src/Raven.Server/Documents/TcpHandlers/SubscriptionConnection.cs

```diff
@@ -142,14 +177,16 @@ public async Task InitAsync()
             AddToStatusDescription(message);
             if (_logger.IsInfoEnabled)
             {
-                _logger.Info(
-                    message);
+                _logger.Info(message);
             }
 
             // first, validate details and make sure subscription exists
             SubscriptionState = await TcpConnection.DocumentDatabase.SubscriptionStorage.AssertSubscriptionConnectionDetails(SubscriptionId, _options.SubscriptionName);
             Subscription = ParseSubscriptionQuery(SubscriptionState.Query);
 
+            _connectionScope.For(SubscriptionOperationScope.Connection).RecordConnectionInfo(SubscriptionState.SubscriptionId, 
```

Comment:
Where do we dispose (close) this:

` _connectionScope.For(SubscriptionOperationScope.Connection)` 

?

Reply:
disposing in:
https://github.com/Danielle9897/ravendb/blob/25d78fcae94323160c198b9202dfc4d9233b3d33/src/Raven.Server/Documents/TcpHandlers/SubscriptionConnection.cs#L1114

Reply:
OK